[SDK-3692] Update readme to match new design (#401)
auth0/express-openid-connect: README.md

Express JS middleware implementing sign on for Express web apps using OpenID Connect.

![Release](https://img.shields.io/npm/v/express-openid-connect)
[![Codecov](https://img.shields.io/codecov/c/github/auth0/express-openid-connect)](https://codecov.io/gh/auth0/express-openid-connect)
![Downloads](https://img.shields.io/npm/dw/express-openid-connect)
[![License](https://img.shields.io/:license-mit-blue.svg?style=flat)](https://opensource.org/licenses/MIT)
![CircleCI](https://img.shields.io/circleci/build/github/auth0/express-openid-connect)

:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](#api-reference) - :speech_balloon: [Feedback](#feedback)

## Documentation

- [Quickstart](https://auth0.com/docs/quickstart/webapp/express) - our guide for quickly adding Auth0 to your Express app.
- [Sample](https://github.com/auth0-samples/auth0-express-webapp-sample/tree/master/01-Login) - an Express app integrated with Auth0.
- [FAQs](./FAQ.md) - Frequently asked questions about express-openid-connect.
- [Docs Site](https://auth0.com/docs) - explore our docs site and learn more about Auth0.

> - [Docs Site(https)
> 

Reviewer: Looks like a left-over.

```suggestion

```

## Getting Started

### Requirements

This library supports the following tooling versions:

> * Node.js `^10.19.0 || >=12.0.0`

Reviewer: Both 10 and 12 are not in LTS anymore, even though we might be compatible, would it make sense to update this to mention the current LTS versions, assuming we support these?

Doesn't have to be in this PR, I just noticed it and wanted to raise it.

Author: Yeah it didn't feel right having 10 and 12 here, but given that we only run tests against 12 currently I didn't want to add 14/16/18 without having at least one of them in CI

Reviewer: Fair point, let's leave this for now.

Reviewer 2: Yeah, that's out of date - will fix that

## Install

> Using [npm](https://npmjs.org) in your project directory run the following command:

Reviewer: ```suggestion
Using [npm](https://npmjs.org) in your project directory, run the following command:
```

Follow our [Secure Local Development guide](https://auth0.com/docs/libraries/secure-local-development) to ensure that applications using this library are running over secure channels (HTTPS URLs). Applications using this library without HTTPS may experience "invalid state" errors.

### Configure Auth0

Create a **Regular Web Application** in the [Auth0 Dashboard](https://manage.auth0.com/#/applications).

> > **If you're using an existing application**, verify that you have configured the following settings in your Single Page Application:

Reviewer: ```suggestion
> **If you're using an existing application**, verify that you have configured the following settings in your Regular Web Application:
```

Follow our [Secure Local Development guide](https://auth0.com/docs/libraries/secure-local-development) to ensure that applications using this library are running over secure channels (HTTPS URLs). Applications using this library without HTTPS may experience "invalid state" errors.

### Configure Auth0

Create a **Regular Web Application** in the [Auth0 Dashboard](https://manage.auth0.com/#/applications).

> > **If you're using an existing application**, verify that you have configured the following settings in your Single Page Application:

Reviewer: It looks like this is copy pasted from a SPA, we should remove the reference to SPA but also ensure the configuration below is correct for a Regular Web App, which might be different from a SPA.

Author: I'd verified this using the sample, just overlooked the SPA reference here, will fix
